tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -444,6 +528,9 @@ void EnqueueWriteBufferCommand::process() {
     } else {
         this->writer.cq_write(this->src, data_size_in_bytes, system_memory_temporary_storage_address);
     }
+    uint32_t data;
+    tt::Cluster::instance().read_sysmem((char*)&data, sizeof(uint32_t), system_memory_temporary_storage_address, 0);
+    uint32_t padded_page_size = align(this->buffer.page_size(), 32);
```

Comment:
what is the purpose of this?

Reply:
oh its deadcode, I used to print out some stuff here.